dashboard/app/models/user.rb

```diff
@@ -1040,6 +1040,14 @@ def upgrade_to_teacher(email, email_preference = nil)
         new_attributes[:email] = email
       end
       update!(new_attributes)
+
+      # Remove family name, in case it was set on the student account.
+      if DCDO.get('family-name-features', false)
+        properties['family_name'] = nil
```

Comment:
nit: if you add `family_name` to `serialized_attrs` you can just have `user.family_name = nil`. Up to you though -- can also be a follow up!

Reply:
oh interesting, thanks! i might do this as a followup just because it'd otherwise require changing/re-running all 3 of the PRs I have up.

Reply:
this will continue to work even if you do add this field to `serialized_attrs`, so definitely fine! Just makes the code a bit more readable.

If it helps [this](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/concerns/serialized_properties.rb) is the implementation of `serialized_attrs` -- it's not super helpful and could use a bit more documentation though